lamindb/_from_values.py

```diff
@@ -276,7 +276,7 @@ def create_records_from_source(
             else {"source": source}
         )
         for bk in bionty_kwargs:
-            records.append(model(**bk, **create_kwargs))
+            records.append(model(**bk, **create_kwargs, _skip_validation=True))
```

Comment:
This modification is the main solution to the search_names problem.

Reply:
Slack thread: https://laminlabs.slack.com/archives/C04FPE8V01W/p1739365084531499